transforms/tabular/tabular-merger-plugin/tests/test_main.py

```diff
@@ -0,0 +1,236 @@
+"""Testing Tabular Merger."""
+import os
+import pathlib
+import random
+import string
+import typing
+
+import filepattern as fp
+import numpy as np
+import pandas as pd
+import pytest
+import vaex
+
+from polus.plugins.transforms.tabular.tabular_merger import tabular_merger as tm
+
+
+class Generatedata:
+    """Generate tabular data with several different file format."""
+
+    def __init__(
+        self,
+        file_pattern: str,
+        outname: str,
+        sameRows: typing.Optional[bool],
+        truncColumns: typing.Optional[bool],
+    ):
+        """Define instance attributes."""
+        self.dirpath = os.path.abspath(os.path.join(__file__, "../.."))
```

Comment:
The `"/"` is not platform agnostic, it will fail on windows. Use `pathlib.Path` instead of `os.path` and avoid using the `"/"`